code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -40,7 +41,6 @@
 					C.take_damage(removed * 2)
 		if(dam)
 			M.adjustToxLoss(target_organ ? (dam * 0.5) : dam)
-			M.add_chemical_effect(CE_TOXIN, removed * strength)
```

Comment:
just replace the adjustToxLoss with CE_TOXIN instead:

- it's how most things already do the whole toxin thing
- it doesn't bypass organ regeneration and interacts with the liver